Getting Started: Review (#1469)
cap-js/docs: about/index.md

#### The Science

- introduces the fundamental concepts, and design principles →  *[Core Concepts](#core-concepts)*
- compares these to related concepts and approaches → *[Related Art#related-art

#### The Art

- applies the science as best and simple as possible → [*Key Features & Qualities*](#key-features-qualities)
- tries to avoid common mistakes as much as possible → *[Bad Practices](#bad-practices)*

#### The Ingredients

The CAP framework features a mix of proven and broadly adopted open-source and SAP technologies. The figure below depicts CAP's place and focus in a stack architecture.

> ![Vertically CAP services are placed between database and UI. Horizontally, CDS fuels CAP services and is closer to the core than, for example, toolkits and IDEs. Also shown horizontally is the integration into vrious platofrm services.](assets/architecture.drawio.svg){style="width:555px"}

Author: ```suggestion
![Vertically CAP services are placed between database and UI. Horizontally, CDS fuels CAP services and is closer to the core than, for example, toolkits and IDEs. Also shown horizontally is the integration into various platform services.](assets/architecture.drawio.svg){style="width:555px"}
```

Following sections provide an overview of the core concepts and design principles of CAP. The illustration below is an attempt to show all concepts, how they relate to each other, and to introduce the terminology.

> ![Service models declare service interfaces, events, facades, and services. Service interfaces are published as APIs and are consumed by clients. Clients send requests which trigger events. Services are implemented in service providers, react on events, and act as facades. Facades are inferred to service interfaces and are views on domain models. Service providers are implemented through event handlers which handle events. also, service providers read/write data which has been declared in domain models. ](assets/concepts.drawio.svg){style="padding-right:50px"}

Author: ```suggestion
![Service models declare service interfaces, events, facades, and services. Service interfaces are published as APIs and are consumed by clients. Clients send requests which trigger events. Services are implemented in service providers, react on events, and act as facades. Facades are inferred to service interfaces and are views on domain models. Service providers are implemented through event handlers which handle events. Also, service providers read/write data which has been declared in domain models. ](assets/concepts.drawio.svg){style="padding-right:50px"}
```

- Services are **agnostic** → *platforms and protocols*{.grey}
- Services are **stateless** → *process passive data*{.grey}

> ![The graphic contains the key design principles as they have been introduced in the surrounding text. In addition the key benefits of these design principles are rapid development, minimized cost, maximum scalability, minimal overhead, late-cut microservices, and grow as you go.](assets/paradigm.drawio.svg)

Author: ```suggestion
![The graphic contains the key design principles as they have been introduced in the accompanying text. In addition the key benefits of these design principles are rapid development, minimized cost, maximum scalability, minimal overhead, late-cut microservices, and grow as you go.](assets/paradigm.drawio.svg)
```
